- Description: zfcp: kernel config text and help change
  Symptom:     Old kernel configuration text and missing
               kernel configuration help text for zfcp
  Problem:     -
  Solution:    Update
  Problem-ID:  -
  Archs:       s390-31, s390-64

--- linux-2.5/drivers/scsi/Kconfig	5 Apr 2004 09:27:13 -0000	1.30
+++ linux-2.5/drivers/scsi/Kconfig	6 May 2004 10:57:57 -0000	1.30.2.1
@@ -1694,8 +1694,17 @@
 #      bool 'Cyberstorm Mk III SCSI support (EXPERIMENTAL)' CONFIG_CYBERSTORMIII_SCSI
 
 config ZFCP
-	tristate "IBM z900 OpenFCP/SCSI support"
+	tristate "FCP host bus adapter driver for IBM eServer zSeries"
 	depends on ARCH_S390 && SCSI
+	help
+          If you want to access SCSI devices attached to your IBM eServer 
+          zSeries by means of Fibre Channel interfaces say Y.
+          For details please refer to the documentation provided by IBM at
+          <http://oss.software.ibm.com/developerworks/opensource/linux390>
+
+          This driver is also available as a module. This module will be 
+          called zfcp. If you want to compile it as a module, say M here 
+          and read Documentation/modules.txt.
 
 endmenu
 

